File: app/src/main/java/com/polok/eubmanagement/presentation/auth/AuthActivity.java
package com.polok.eubmanagement.presentation.auth;

import android.os.Bundle;
import com.polok.eubmanagement.base.BaseActivity;
import com.polok.eubmanagement.databinding.ActivityAuthBinding;
import com.polok.eubmanagement.util.Extension;
import com.polok.eubmanagement.widget.PrimaryLoader;

public class AuthActivity extends BaseActivity<ActivityAuthBinding> {
    @Override
    protected ActivityAuthBinding initViewBinding() {
        return ActivityAuthBinding.inflate(getLayoutInflater());
    }

    @Override
    protected void initOnCreateView(Bundle savedInstanceState) {
        Extension.hideStatusBar(getWindow());
    }
}

File: app/src/main/java/com/polok/eubmanagement/presentation/auth/AuthActivity.kt
package com.polok.eubmanagement.presentation.auth

import android.os.Bundle
import com.polok.eubmanagement.R
import com.polok.eubmanagement.base.BaseActivity
import com.polok.eubmanagement.databinding.ActivityAuthBinding
import com.polok.eubmanagement.util.getColorCompat

class AuthActivity : BaseActivity<ActivityAuthBinding>(
    viewBindingFactory = ActivityAuthBinding::inflate
) {
    override fun initOnCreate(savedInstanceState: Bundle?) {
        window.statusBarColor = resources.getColorCompat(R.color.color_window)
    }

    override fun onDestroy() {
        super.onDestroy()
        window.statusBarColor = resources.getColorCompat(R.color.color_primary)
    }
}